storage/simulation/store.go

```diff
@@ -87,3 +93,14 @@ func (s *Store) String(rangeCount int) string {
 		desc.StoreID, desc.Node.NodeID, desc.Capacity.RangeCount, desc.Capacity.Available/bytesPerRange,
 		desc.Capacity.Capacity, desc.Capacity.Available)
 }
+
+// GossipStore broadcasts the store on the gossip network.
+func (s *Store) gossipStore(rangeCount int) error {
+	s.RLock()
+	defer s.RUnlock()
+	desc := s.getDesc(rangeCount)
```

Comment:
yep, and perfectly fine again, double read lock is fine.  But I'll remove all locks in an upcoming PR.